app/assets/stylesheets/Login/Login.scss

```diff
@@ -0,0 +1,87 @@
+@import "../home/components/variables";
+
+body {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+}
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
```

Comment:
you could refactor this code creating a group of properties to extend whenever you need like this

```
%flex center {
  display: flex;
  align-items: center;
  justify-content: center;
}

.form-container {
  @extend %flex-center;
}
```

and avoid to re-write those properties.

you can check more info about this feature here https://sass-lang.com/guide